docs/resources/modelarts_service.md

```diff
@@ -0,0 +1,369 @@
+    + **edge**: An edge service, which uses Intelligent EdgeFabric (IEF) to deploy a model as a web service on an edge
+                node created on IEF.
+
+  Changing this parameter will create a new resource.
+
+* `config` - (Required, List) Model running configurations.  
+  If **infer_type** is **batch** or **edge**, you can configure only one model.
+  If**infer_type**is**real-time**, you can configure multiple models and assign weights based on service requirements.
+  However, the versions of multiple models must be unique.
+  The [Config](#ModelartsService_Config) structure is documented below.
+
+* `workspace_id` - (Optional, String, ForceNew) ID of the workspace to which a service belongs.  
+  The default value is **0**, indicating the default workspace.
+
+  Changing this parameter will create a new resource.
+
+* `description` - (Optional, String) The description of the service.  
+
+* `pool_name` - (Optional, String, ForceNew) The ID of the new dedicated resource pool.  
+  When using dedicated resource pool to deploy services, ensure that the cluster status is normal.
+  If both **pool_name** and **config.pool_name** are configured, **pool_name** in real-time config is preferred.
+
+  Changing this parameter will create a new resource.
+
+* `vpc_id` - (Optional, String, ForceNew) The VPC ID to which a real-time service instance is deployed.  
+  By default, this parameter is left blank. In this case, ModelArts allocates a dedicated VPC to each user,
+  and users are isolated from each other.
+  To access other service components in the VPC of the service instance,
+  set this parameter to the ID of the corresponding VPC.
+  Once a VPC is configured, it cannot be modified. If both vpc_id and pool_name are configured,
+  only the dedicated resource pool takes effect.
+
+  Changing this parameter will create a new resource.
+
+* `subnet_id` - (Optional, String, ForceNew) The subnet ID.  
+  By default, this parameter is left blank.
+  This parameter is mandatory if vpc_id is configured.
+  Enter the network ID displayed in the subnet details on the VPC management console.
+  A subnet provides dedicated network resources that are isolated from other networks.
+
+  Changing this parameter will create a new resource.
+
+* `security_group_id` - (Optional, String, ForceNew) The security group ID.  
+  By default, this parameter is left blank.
+  This parameter is mandatory if vpc_id is configured.
+  A security group is a virtual firewall that provides secure network access control policies for service instances.
+  A security group must contain at least one inbound rule to permit the requests whose protocol is TCP,
+  source address is 0.0.0.0/0, and port number is 8080.
+
+  Changing this parameter will create a new resource.
+
+* `schedule` - (Optional, List) Service scheduling configuration, which can be configured only for real-time services.  
+  By default, this parameter is not used. Services run for a long time.
```

Comment:
how should I understand about `this parameter is not used`?

Reply:
fixed